docs/css/ez-guidelines.css

```diff
@@ -45,3 +48,12 @@
 .ez-guidelines-sample__correct-block .ez-icon {
     margin-top: 1em;
 }
+/* Spacing between rendered example and code block*/
+.ez-guidelines.md-typeset > .codehilitetable,
```

Comment:
Wouldn't it be better to create a special class for adding top spacing instead?

Reply:
I agree @sunpietro. I did it (`ez-guidelines-utilities.css`), because Bootstrap 3 doesn't cover offset classes. The issue is that that specific class (I'm referring to `.codehilitetable`), for instance, is an attribute selector linked to the Syntax examples. This was the way I decided given the constraints.